src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -9,6 +9,118 @@
 
 namespace System.Buffers
 {
+
+    public static class MemoryListExtensions
```

Comment:
Are we going to use the ReadableBuffer implementation?

Reply:
For? 

BTW, this project is not even referenced by the pipelines. I needed to add all these extension methods below to be able to compile existing code (in this project) after removing CopyTo, and ISequence implementations from IMemoryList. i.e. there is no plan to ship these members.

Reply:
Ah I see